support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx

```diff
@@ -239,9 +269,11 @@ export function SummaryTsAndCs({
 	productKey,
 	cssOverrides,
 }: SummaryTsAndCsProps): JSX.Element {
+	const inAdLite = productKey === 'GuardianAdLite';
 	const inTier3 = productKey === 'TierThree';
 	const inTier2 = productKey === 'SupporterPlus';
-	const inTier1 = productKey === 'Contribution' || !(inTier2 || inTier3);
+	const inTier1 =
+		productKey === 'Contribution' || !(inTier2 || inTier3 || inAdLite);
```

Comment:
Should we rename these? The tier model seems a bit out of step with the ad-lite product

Reply:
Ok, have done in this file to see if works (think throughout would need to be a new PR and probably run it by George). `inTier1` => `inSupport`, `inTier2` => `inAllAccessDigital`, `inTier3` => `inDigitalPlusPrint`